packages/amplify-cli/src/commands/init.ts

```diff
@@ -18,11 +18,12 @@ const constructExeInfo = (context: $TSContext): void => {
   };
 };
 
+const isHeadless = true;
```

Comment:
I would name this input `recommendGen2: true|false`

Reply:
hmmmm.... that makes sense for this PR, (might be unlikely) but what if we add something else to `preInitSetup` not Gen2 related?

Reply:
If we're worried about that, we can pass in an options object `{recommendGen2: boolean}`. However, since this is an internal function, that could always be a future refactor if there are more options later on

Reply:
Done dfba32eb5086bd4918c3daebe949ca439c49f543